src/isotools/gene.py

```diff
@@ -834,8 +834,8 @@ def coordination_test(self, samples=None, test: Literal['fisher', 'chi2'] = "fis
             The samples can be provided either as a single group name, a list of sample names, or a list of sample indices.
         :param test: Test to be performed. One of ("chi2", "fisher")
         :type test: str
-        :param min_dist: Minimum distance (in nucleotides) between the two alternative splicing events for the pair to be tested.
-        :type min_dist: int
+        :param min_dist_AB: Minimum distance (in nucleotides) between node A and B in an event
+        :param min_dist_events: Minimum distance (in nucleotides) between the two alternative splicing events for the pair to be tested.
```

Comment:
Maybe a better description?
Minimum distance (in nucleotides) between all pairs of corresponding nodes in the two alternative splicing events being tested.